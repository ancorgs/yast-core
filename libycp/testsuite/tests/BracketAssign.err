Parsed:
----------------------------------------------------------------------
{
  map `m = $[1:11, "a":"aa", `b:`bb];
  map `m1 = m;
  integer `i = 12;
  `m[1] = i;
  `i = (13);
  return (m == m1);
}
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
{
  map `m = $[1:11, "a":"aa", `b:`bb];
  map `m1 = eval (m);
  integer `i = 12;
  `m[1] = i;
  `i = (13);
  return (m == m1);
}
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
{
  map `m = $[1:11, "a":"aa", `b:`bb];
  `m[2] = 12;
  return m;
}
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
{
  list `l = [1, "a", `b];
  `l[1] = 2;
  return l;
}
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
{
  list `l = [1, "a", `b];
  `l[3] = 33;
  return l;
}
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
{
  list `l = [1, "a", `b];
  `l[8] = 33;
  return l;
}
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
{
  list `l = [1, "a", `b];
  map `m = $[1:l, "a":l, `b:l];
  `m["a", 1] = 11;
  return [l, m];
}
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
{
  list `l = [1, "a", `b];
  map `m = $[1:l, "a":l, `b:l];
  `m["a"] = [2, "b", `c];
  return m;
}
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
{
  list `l = [1, "a", `b];
  map `m = $[1:l, "a":l, `b:l];
  `m["b"] = "This is b";
  return m;
}
----------------------------------------------------------------------
