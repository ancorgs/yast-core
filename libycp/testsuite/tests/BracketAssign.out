(true)
(false)
($[1:11, 2:12, "a":"aa", `b:`bb])
([1, 2, `b])
([1, "a", `b, 33])
([1, "a", `b, nil, nil, nil, nil, nil, 33])
([[1, 11, `b], $[1:[1, 11, `b], "a":[1, 11, `b], `b:[1, 11, `b]]])
($[1:[1, "a", `b], "a":[2, "b", `c], `b:[1, "a", `b]])
($[1:[1, "a", `b], "a":[1, "a", `b], "b":"This is b", `b:[1, "a", `b]])
